src/emonesp.h

```diff
@@ -5,15 +5,7 @@
 // General support code used by all modules
 // -------------------------------------------------------------------
 
-// Uncomment to use hardware UART 1 for debug else use UART 0
-// #define DEBUG_SERIAL1
```

Comment:
Yep, it's just that the default build seems to be outputting debug on the main serial line, which is a bug IMHO.